adk/ADK.py

```diff
@@ -64,20 +65,17 @@ def wrap_binary_data(self, data):
 
     def format_response(self, response):
         if self.is_binary(response):
-            content_type = 'binary'
-            response = base64.b64encode(response)
-            if not isinstance(response, six.string_types):
-                response = str(response, 'utf-8')
-        elif isinstance(response, six.string_types) or isinstance(response, six.text_type):
-            content_type = 'text'
+            content_type = "binary"
+            response = str(base64.b64encode(response), "utf-8")
+        elif isinstance(response, six.string_types) or isinstance(
+            response, six.text_type
+        ):
+            content_type = "text"
         else:
-            content_type = 'json'
-        response_string = json.dumps({
-            'result': response,
-            'metadata': {
-                'content_type': content_type
-            }
-        })
+            content_type = "json"
+        response_string = json.dumps(
```

Comment:
I agree with simplifying this line, but why  not just have this all on a single line? I typically do expand our JSON formatting, but nothing else.

Reply:
This one was again `black`'s opinion, happy to do your way.